src/constants/index.ts

```diff
@@ -1,5 +1,4 @@
 import { CloudPlatform } from 'generated/tdr';
-import _ from 'lodash';
```

Comment:
I'm surprised you can remove the import from lodash without any other changes to the file. Do you have any linting for unused imports?

Reply:
Good point! Our eslint rules don't include unused imports so I created a ticket for that: https://broadworkbench.atlassian.net/browse/DR-3176